Source/Core/Core/IOS/IOS.h

```diff
@@ -82,6 +83,12 @@ enum ProcessId : u32
   PID_UNKNOWN = 19,
 };
 
+template <typename T>
+void WriteReturnValue(T value, u32 address)
```

Comment:
It could be put somewhere else (where?).

I do like having a dedicated statement for return values to separate them from other random data - they are all really s32 and share intent.